themes/shijin4/layouts/partials/home.html

```diff
@@ -25,17 +25,17 @@
 				<div class="loader"><span>&#8635;</span></div>
 				<div style="text-align:right;margin-top:4px"><a target="_blank" href="https://dev.haiku-os.org/timeline">more</a></div>
 			</div>
-			<div class="tab-pane" id="ml">
-				<div class="loader"><span>&#8635;</span></div>
-				<div style="text-align:right;margin-top:4px"><a target="_blank" href="https://www.freelists.org/archive/haiku-development/">more</a></div>
-			</div>
 			<div class="tab-pane" id="pkgs">
 				<div class="loader"><span>&#8635;</span></div>
 				<div style="text-align:right;margin-top:4px"><a target="_blank" href="https://depot.haiku-os.org/#!/?viewcrttyp=MOSTRECENT">more</a></div>
 			</div>
+			<div class="tab-pane" id="ml">
+				<div class="loader"><span>&#8635;</span></div>
+				<div style="text-align:right;margin-top:4px"><a target="_blank" href="https://www.freelists.org/archive/haiku-development/">more</a></div>
+			</div>
 			<div class="tab-pane" id="nightlies">
-				<p>The Haiku source is continually built and released for testing purposes nearly every day. You can download and install these latest snapshots to check out the latest features and bug-fixes.</p>
-				<p>Be aware that nightly images may be unstable. Additionally, some packages included with official releases need to be installed separately.</p>
+				<p>New builds of Haiku are being constantly built and released for testing purposes daily. You can download and install these latest snapshots to check out the latest features and bug fixes.</p>
```

Comment:
The wording here was not great but I don't find your version a lot better. Also, you say daily and then happily switch to nightly, which, ok, we all live in different timezones, but...